website/blog/2023-03-02-ferretdb-building-mongodb-compatible-database.md

```diff
@@ -0,0 +1,94 @@
+While there is a wide selection of open source document databases, none of them are compatible with MongoDB.
+
+This is why we founded [FerretDB](https://www.ferretdb.io/): we are on a mission to bring MongoDB database workloads back to its open source grounds.
+We are enabling [PostgreSQL](https://www.postgresql.org/) and other database backends to run MongoDB workloads, retaining the opportunities provided by the existing ecosystem around MongoDB.
+
+## So where are we now?
+
+A bit more than a year after we started working on FerretDB full time, we are a team of 9 people, working full-time and remotely from all over the world.
+We are executing on our [roadmap](https://github.com/orgs/FerretDB/projects/2), and are planning to release our 1.0 GA by the end of March.
+
+## Where are we headed?
+
+We would like to create a new standard for document databases, with [MongoDB compatibility](https://blog.ferretdb.io/mongodb-compatibility-whats-really-important/).
+FerretDB is a drop-in replacement for MongoDB, but it also aims to set a new standard which not only brings easy to use document databases back to its open source roots, but also enables different relational database engines to run document database workloads in a standardized way.
+
+Also, we are building an easy to use database offering the flexibility of document databases, with the possibility to run queries on the same database using SQL.
+
+## What is the FerretDB GA going to be capable of?
+
+FerretDB 1.0 GA includes all the essential features capable of running document-type workloads.
+
+We are testing FerretDB 0.9.2 against real-world applications, like [BigBlueButton](https://bigbluebutton.org/), [Strapi](https://strapi.io/), or frameworks such as [MeteorJS](https://www.meteor.com/), among others.
+We also confirmed that popular database management software such as [Mongosh/MongoDB Compass](https://www.mongodb.com/docs/compass/current/embedded-shell/), [NoSQL Booster](https://nosqlbooster.com/downloads), [Mingo](https://mingo.io/) are able to leverage the current feature set of FerretDB.
+It’s like managing a MongoDB database, but it is FerretDB (and open source) under the hood.
+We think this is insanely cool!
+
+## What database backends does FerretDB support?
+
+FerretDB offers a fully pluggable architecture, we support other backends as well, and these can be contributed by the community.
+
+### PostgreSQL
+
+We are building FerretDB on PostgreSQL and we envision that this is going to be our main database backend in the foreseeable future.
+This is the backend which will get the newest features of FerretDB.
+We are implementing features adding MongoDB compatibility to Postgres mainly using its JSON capabilities.
+However, we recognize that we need to depart from this approach to increase performance by creating our own extension or through other methods.
```

Comment:
Hmm, so we no longer want to use JSON? I understand increasing performance is important to us, but not seeing the relationship with JSON. :thinking:

Reply:
This one need clarifying, do you mean that the performance is prioritized over compatibility?